src/Angor/Client/Pages/Investor.razor

```diff
@@ -262,58 +248,46 @@
         {
             Stats.Clear();
 
-            foreach (var project in projects)
-            {
-                var projectStats = await _IndexerService.GetProjectStatsAsync(project.ProjectInfo.ProjectIdentifier);
-                Stats.Add(project.ProjectInfo.ProjectIdentifier, projectStats);
-            }
+            var tasks = projects.Select(x => AddProjectStats(x.ProjectInfo.ProjectIdentifier));
+            await Task.WhenAll(tasks);
         }
         catch (Exception ex)
         {
             notificationComponent.ShowErrorMessage(ex.Message);
         }
     }
 
-    public string TrimString(string input)
```

Comment:
no replacement to this method or was it not used?

Reply:
Not used